lang/py3/avro/datafile.py

```diff
@@ -403,10 +403,6 @@ def __exit__(self, type, value, traceback):
   def __iter__(self):
     return self
 
-  def __next__(self):
```

Comment:
Did not know that Python would allow having a method implemented twice 😱

Reply:
It's basically just a dictionary. One eats the next one. :(

```python
>>> class foo(object):
...   def hello(self):
...     print('i am hello1')
...   def hello(self):
...     print('i am hello2')
>>> import pprint
>>> pprint.pprint(foo.__dict__)
mappingproxy({'__dict__': <attribute '__dict__' of 'foo' objects>,
              '__doc__': None,
              '__module__': '__main__',
              '__weakref__': <attribute '__weakref__' of 'foo' objects>,
              'hello': <function foo.hello at 0x7f450cd85f28>})
```